### DISCUSSION TITLE
。

### DISCUSSION THREAD

**Discussion by ghost** [anuraghazra/github-readme-stats#2562] [Q&A] [answered]
Can I add private repos stats to the Top Langs

**Comment by Zo-Bro-23** [comment, marked as answer]
> We don't have access to private dependencies on the Public Vercel instance. The count_private parameter exists on the stats card because people can share the number of private commits on their GitHub profile.
>
> ![image](https://user-images.githubusercontent.com/62234360/222752356-84da01c5-56fe-47fc-8965-e5d33295fed3.png)
>
> If you want your private contributions to show up on the language card, you can deploy the Vercel instance yourself (See https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).

See #2478 for more.
